src/chi/components/modal/modal.scss

```diff
@@ -7,6 +7,15 @@ $animation-duration: 0.5s;
   overflow: hidden;
 }
 
+.-long-content-aligner {
```

Comment:
Doubting about this class name

Reply:
@jllr If -disableScroll is camel case, -long-content-aligner should be as well, or the opposite.